src/async.rs

```diff
@@ -147,6 +147,20 @@ impl AsyncClient {
         Ok(resp.error_for_status()?.json().await?)
     }
 
+    /// Get the serialized [`Block`] given a particular [`BlockHash`].
+    pub async fn get_block_raw(&self, block_hash: &BlockHash) -> Result<Option<Block>, Error> {
```

Comment:
As the returned block is not *raw* anymore (e.g., a `Vec<u8>`), I'd suggest to simply name this `get_block` or `get_block_{by/for}_hash`?